docs-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs-app",
-  "version": "0.0.1-beta.0",
+  "version": "0.0.1-beta.1",
```

Comment:
Will this try to cut an npm release for the docs app? Looks like the package name is just `docs-app`, so not sure if we want that or not?

Reply:
Hm, no, it's [private](https://github.com/CrowdStrike/ember-headless-form/blob/7aae1f3c2aeeb4f7608b21eca603a37fd7b7bdff/docs-app/package.json#L4), so that shouldn't do anything (otherwise that would be really bad).

I guess it bumps the version _internally_, because its dependencies have changed also, given that the addon was updated. That's at least how I read its own changelog entry: https://github.com/CrowdStrike/ember-headless-form/pull/75/files#diff-39fb47973af070f15e91dd3a8a0cd60c51d42a5036449f6287bca7a6d49139ea

Reply:
Ah okay that makes sense. Thanks!

Reply:
Personally, i'd add docs-app to here: https://github.com/CrowdStrike/ember-headless-form/blob/main/.changeset/config.json#L17

So that the tool has less to do.